examples/recompute/bert/run_classifier.py

```diff
@@ -112,7 +112,7 @@ def evaluate(exe, test_program, test_pyreader, fetch_list, eval_phase):
     time_begin = time.time()
     for data in test_pyreader():
         np_loss, np_acc, np_num_seqs = exe.run(program=test_program,
-						   feed=data,
+         feed=data,
```

Comment:
Is there a pre-commit to format them?

Reply:
Currently not for fleet, will update this later.